From 120c708d51f72ade4a31d3d8f35bcfad7b12e723 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 6 Sep 2022 20:38:03 -0700
Subject: [PATCH] civetweb: Disable lto

lto does not work everywhere, therefore disable it atleast in the cmake
file, we can still enable it via environment

Upstream-Status: Pending

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 thirdparty/civetweb-1.10/CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

--- a/thirdparty/civetweb-1.10/CMakeLists.txt
+++ b/thirdparty/civetweb-1.10/CMakeLists.txt
@@ -298,7 +298,6 @@ add_c_compiler_flag(/WX)
 add_c_compiler_flag(-pedantic-errors)
 add_c_compiler_flag(-fvisibility=hidden)
 add_c_compiler_flag(-fstack-protector-strong RELEASE)
-add_c_compiler_flag(-flto RELEASE)
 
 add_c_compiler_flag(-fstack-protector-all DEBUG)
 if (MINGW)
@@ -361,7 +360,6 @@ if (CIVETWEB_ENABLE_CXX)
   add_cxx_compiler_flag(-pedantic-errors)
   add_cxx_compiler_flag(-fvisibility=hidden)
   add_cxx_compiler_flag(-fstack-protector-strong RELEASE)
-  add_cxx_compiler_flag(-flto RELEASE)
 
   add_cxx_compiler_flag(-fstack-protector-all DEBUG)
   if (MINGW)
